src/utils/extractWhiteboard.js

```diff
@@ -0,0 +1,30 @@
+const extractWhiteboardTags = whiteboard => {
```

Comment:
You can instead do

```js
export default (whiteboard) => {
 // ...
};
```

After that you can delete ` export default extractWhiteboardTags; `